themes/mongodb/src/js/componentThirdParty.js

```diff
@@ -1,20 +1,4 @@
-const SAMPLE_FACTORS = {
-    'charts': 0.12158,
-    'guides': 0.02644,
-    'stitch': 0.03162,
-    'docs-ruby': 0.47984,
-    'ecosystem': 0.00997,
-    'docs-php-library': 0.04165,
-    'atlas': 0.01176,
-    'compass': 0.01684,
-    'manual': 0.00022,
-    'landing': 0.00806,
-    'mongoid': 0.07576,
-    'mms-cloud': 0.06358,
-    'mms-onprem': 0.01790,
-    'bi-connector': 0.03912,
-    'spark-connector': 0.05562
-};
+const ENABLED_SITES_FOR_DELIGHTED = new Set(['docs', 'guides', 'manual']);
```

Comment:
What URLs does 'docs' correspond to here?

Reply:
The `docs` repo will be changing its project name from `manual` to `docs` after it is integrated with snooty (so it would map to https://docs.mongodb.com/manual/). I was gonna leave in both for now until everything was all set and changed.

Reply:
Can we report https://docs.mongodb.com/manual/ URLs as `manual` in Delighted?